AGPs/AGP-CS.md

```diff
@@ -0,0 +1,64 @@
+---
+AGP: N/A
+Title: Develop and launch Aragon Chain
+Author: Aidan Hyman & Gregory Markou
+Status: Stage III
+Track: Finance
+Created: 2019-10-03
+---
+
+# Develop and launch Aragon Chain
+
+## Address of the transfer recipient
+
+0xfc6eBB0F689674b18f90dA73658072155e98ea5B
+
+## Amount of the transfer
+
+A total of $x, will be allocated to the Aragon Chain. Invoices will be sent to the Aragon Association, and compensation will be at their discretion.
```

Comment:
Please don't forget to change `x` for the final amount before the proposal needs to be frozen